test/installer/scripts/prep_omsadmin.sh

```diff
@@ -4,23 +4,25 @@ RUBY_TESTVERS=$2
 TESTDIR_SKEL=/tmp/test_omsadmin_
 TESTDIR=$TESTDIR_SKEL$$
 mkdir -p $TESTDIR
+
 echo "$TESTDIR"
 
 OMSADMIN=$TESTDIR/omsadmin.sh
 cp $BASE_DIR/installer/scripts/omsadmin.sh $OMSADMIN
 chmod u+wx $OMSADMIN
 
-sed -i s,VAR_DIR=.*,VAR_DIR=$TESTDIR,1 $OMSADMIN
```

Comment:
I think we need to update the system tests to be multi-homing aware. For example, the reonboard test is wrong because the 2 workspaces shared the same location. I'll create a task to track this.